npm.js

```diff
@@ -81,6 +81,7 @@ var NPMLocation = function(options, ui) {
   this.registryURL = options.registry || defaultRegistry;
   this.tmpDir = options.tmpDir;
   this.remote = options.remote;
+  this.strictSSL = options.strictSSL;
```

Comment:
If we add the `strictSSL in options ? options.strictSSL : true` check here, then this feature can be backwards-compatible and not a new release of jspm-npm.